docker/Dockerfile

```diff
@@ -165,7 +165,7 @@ WORKDIR /home/blueoil
 
 # For development
 RUN apt-get update && apt-get install -y \
-     x11-apps imagemagick
+     x11-apps
 
 RUN  apt-get clean && \
```

Comment:
Could you remove this line also?

Reply:
You Mean the from line 166-168 right?

Reply:
I meant line 166-171 in the original file.